tron/core/actionrun.py

```diff
@@ -442,7 +454,27 @@ def fail(self, exit_status=0):
 
         return self._done('fail', exit_status)
 
+    def emit_triggers(self):
+        if isinstance(self.trigger_downstreams, bool):
+            shortdate = self.render_template("{shortdate}")
+            triggers = [f"shortdate.{shortdate}"]
+        elif isinstance(self.trigger_downstreams, dict):
+            rendered = [
+                (k, self.render_template(v))
```

Comment:
We should have validation for these templates.

Reply:
should probably belong in config validation, like the command attribute?